src/seq/index.rs

```diff
@@ -31,9 +31,17 @@ pub enum IndexVec {
 impl IndexVec {
     /// Returns the number of indices
     pub fn len(&self) -> usize {
-        match self {
-            &IndexVec::U32(ref v) => v.len(),
-            &IndexVec::USize(ref v) => v.len(),
+        match *self {
+            IndexVec::U32(ref v) => v.len(),
+            IndexVec::USize(ref v) => v.len(),
+        }
+    }
+
+    /// Returns `true` if the length is 0.
+    pub fn is_empty(&self) -> bool {
```

Comment:
Does Clippy also recommend increasing the API surface?

I don't mind this addition, but generally prefer API minimality.

Aside: I think these functions should have `#[inline]`?

Reply:
Yes, this is recommended by clippy and mirrors `std`. `is_empty` is supposed to be clearer and more performant.

I'll add `#[inline]`.